Epsilon.Host.WebApi/Controllers/DocumentController.cs

```diff
@@ -29,7 +29,7 @@ public async Task<ActionResult<PageComponent>> GetPage(int courseId, string page
     [HttpPost("page/{pageName}")]
     public async Task<ActionResult<PageComponent>> UpdateOrCreatePage(int courseId, string pageName, [FromBody] PageUpdateRequest updateRequest)
     {
-        var createdPageComponent = await _pageComponentManager.CreateOrUpdate(courseId, pageName, updateRequest.Body);
+        var createdPageComponent = await _pageComponentManager.CreateOrUpdate(courseId, pageName, updateRequest.Body ?? "");
```

Comment:
Maybe instead of assigning a default value to the body, make it required using the `[Required]` attribute. This will cause the API to return a 400 error indicating a bad request. That makes it clearer that the body was null, and that it probably shouldn't be. Then again a null body basically means an empty string, so maybe its not that important. Up to you.